### ISSUE TITLE
Move handbook meta info inside complementary landmark

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/wporg-developer#400]
Would it be possible to also move the meta information inside the complementary landmark? It would likely be more discoverable just above the Chapters heading.

_Originally posted by @alexstine in https://github.com/WordPress/wporg-developer/issues/368#issuecomment-1821529465_

**Comment by outdoor2kode** [comment]
@adamwoodnz, would you mind sharing more context here? What does the 'meta information' mentioned here refer to?
Thanks 🙏

**Comment by adamwoodnz** [comment]
Sure, the request is to move the handbook article meta info from the bottom of the article into the complementary landmark, ie. the container for the Chapter List. If we make this change in the markup, visually it would mean moving from 

![Screenshot 2024-01-29 at 4 10 41 PM](https://github.com/WordPress/wporg-developer/assets/1017872/ec4cfae1-3731-4d54-a983-513212411c8a)

to

![Screenshot 2024-01-29 at 4 09 52 PM](https://github.com/WordPress/wporg-developer/assets/1017872/2e7f9046-0f07-4959-b250-96fb67e8ea0f)

If this isn't a desirable visual change we could duplicate the content and use the `screen-reader-text` class in the sidebar, and `aria-hidden` in the footer.

@WordPress/meta-design what do you think about the request please?

**Comment by jasmussen** [comment]
I think we can probably move it in there. How would it look if the three columns just became rows? If need be we can refine the prose a bit.

**Comment by fcoveram** [comment]
Agree we can move it.

In addition to making them rows, I would add the TOC divider above the "First published item with the same spacing values to split the area visually.

<img src="https://github.com/WordPress/wporg-developer/assets/895819/89caf5a1-ce02-44a2-9374-3cacaea7c2bb" alt="Table of content with divider highlighted" width="250">

**Comment by outdoor2kode** [comment]
>  It would likely be more discoverable just above the Chapters heading.

I noticed Alex initially suggested placing the meta info above the Chapters heading. I'm curious about the potential impact if we place it on the bottom of the Chapter List as it's more visually coherent. cc @alexstine

> How would it look if the three columns just became rows?

Does it make sense to put it on top of the collapsed chapter list dropdown?
Or on the bottom of the chapter list (consistent with the relative position on the desktop view)
cc @WordPress/meta-design 

![Screenshot 2024-01-30 at 00 34 05](https://github.com/WordPress/wporg-developer/assets/18050944/7e7961bd-8fcb-4202-bdf8-6ba3f06b75ba)

**Comment by adamwoodnz** [comment]
> I noticed Alex initially suggested placing the meta info above the Chapters heading. I'm curious about the potential impact if we place it on the bottom of the Chapter List as it's more visually coherent.

I agree it would look better below the chapters when positioned to the side. We could place it above in the markup though and use flex order to place it below visually.

**Comment by alexstine** [comment]
@adamwoodnz Doing this would cause an WCAG failure. You are unable to move this information visually because of the tabbable link.

> Improve it on GitHub: Contributor Guide”

https://developer.wordpress.org/block-editor/contributors/

Essentially, the focus indicator would be lost when the link is focussed wrapped in the `screen-reader-text`.

Thanks.
